### DISCUSSION TITLE
How do detectors work?

### DISCUSSION THREAD

**Discussion by jumpjack** [ricktu288/ray-optics#133] [General]
What am I reading here?

![image](https://github.com/ricktu288/ray-optics/assets/1620953/096502a5-cde7-4bc7-b550-b06273ddcd12)

**Comment by ricktu288** [comment]
P: Rate of energy flow through the line segment
F⊥: Rate of perpendicular momentum flow through the line segment
F∥: Rate of parallel momentum flow through the line segment

When "ray density" is high enough, the irradiance map will be shown in blue region (in what you show it is too low so only some white lines).
